apple/testing/default_runner/ios_test_runner.bzl

```
@@ -54,8 +54,8 @@ def _ios_test_runner_impl(ctx):
             test_environment = ctx.attr.test_environment,
         ),
         DefaultInfo(
-            runfiles = ctx.runfiles(
-                files = [ctx.file._testrunner],
+            runfiles = ctx.runfiles().merge(
+                ctx.attr._testrunner[DefaultInfo].default_runfiles,
             ),
```

```suggestion
            runfiles = ctx.attr._testrunner[DefaultInfo].default_runfiles,
```

would this be the same?

---

same for others if so

---

I'm not entirely sure -- especially since `ctx.runfiles` is a method on `ctx` rather than a bare constructor

---

https://docs.bazel.build/versions/master/skylark/lib/ctx.html#runfiles this makes it sound like it has nothing by default because "Creates a runfiles object."